skbio/stats/ordination/_principal_coordinate_analysis.py

```diff
@@ -77,22 +90,39 @@ def pcoa(distance_matrix):
        appear, allowing the user to decide if they can be safely
        ignored.
     """
-    distance_matrix = DistanceMatrix(distance_matrix)
+    distance_matrix_obj = DistanceMatrix(distance_matrix)
 
-    E_matrix = e_matrix(distance_matrix.data)
+    # Center distance matrix, a requirement for PCoA here
+    distance_matrix = center_distance_matrix_optimized(
```

Comment:
Assuming this is the updated inplace operation, then it would be imperative to note this in the changelog. If this is inplace, then it silently breaks the API as the existing code did not operate inplace. This can very easily lead to bugs in callers that are using older versions of skbio. Would it be possible to add in a kwarg for using inplace or not?

Reply:
Excellent, excellent catch. Thank you.

Reply:
Yes, thanks

Reply:
Ok, inplace made a kwarg, `False` by default. Also added an additional unit test for `center_distance_matrix` method -- one tests inplace and one non-inplace